cmd/kyverno/main.go

```diff
@@ -624,12 +637,17 @@ func main() {
 		dClient,
 		openApiManager,
 	)
+	cacheResolvers := engineContext.NewResolverChain(
+		engineContext.NewInformerBasedResolver(kubeResourceInformer.Core().V1().ConfigMaps().Lister()),
```

Comment:
You must get the informer before start/wait for cache sync.

Reply:
We are getting Lister here. We are creating informer on line 479

Reply:
Line 479 is setting up the factory.

Reply:
fixed